application/src/main/java/org/togetherjava/tjbot/commands/mathcommands/TeXCommand.java

```diff
@@ -36,6 +36,7 @@
 public class TeXCommand extends SlashCommandAdapter {
 
     private static final String LATEX_OPTION = "latex";
+    private static final Pattern LATEX_PATTERN = Pattern.compile("(\\$\\S+\\$)|([a-zA-Z!., ]+)");
```

Comment:
`LATEX_PATTERN` is not really a good name. It should describe more what it is actually about. It is your inline latex replacement pattern.